tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/textfieldselection-setSelectionRange.html

```diff
@@ -114,6 +114,18 @@
     assert_equals(input.selectionStart, 0, "element.selectionStart should be 0");
     assert_equals(input.selectionEnd, 1, "element.selectionEnd should be 1");
   },'input setSelectionRange(undefined,1)');
+
+  test(function() {
+    var q = false;
+    input.onselect = this.step_func_done(function(e) {
+      assert_true(q, "event should be queued");
+      assert_true(e.isTrusted, "event is trusted");
+      assert_false(e.bubbles, "event bubbles");
```

Comment:
Yes, you're right. I'm quite surprised that when I run this locally it worked fine.